src/config.cpp

```diff
@@ -124,6 +124,12 @@ Variables readVariables(const nlohmann::json& json) {
     return variables;
 }
 
+std::string toAbsolute(const fs::path& base, const std::string& pathStr) {
+    const fs::path path(pathStr);
```

Comment:
```suggestion
std::string toAbsolute(const fs::path& base, const fs::path& path) {
```
seems cleaner (and more flexible in case you want to call this with two arguments that are already paths)

Reply:
Agree, its better than mixing path and string arguments, thanks!